### ISSUE TITLE
Add the new entire thread takedown log to the list of tracker spreadsheets linked from the review call agenda

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#1606]
This log is new. 

I've just come across it and linked to it from 

https://wdtkwiki.mysociety.org/wiki/Main_Page

Related:

* https://github.com/mysociety/alaveteli/issues/1203

**Comment by HelenWDTK** [comment]
This is something I'm doing informally, so I've got no plans to add this.

**Comment by RichardTaylor** [comment]
What does informally mean? 

Does it mean not comprehensive? No confirmed intent to continue the log? 

I think it would be useful to discuss if the log should be kept, what should be included, and if/how it is reviewed.  I have already added new columns for recording any consideration of journalistic value in the material deleted, and if a substantive FOI request was involved or not (the latter was being recorded as part of a free-text notes field). 

I have found the log useful to identify cases of deleted material (eg. those mentioned, but not identified, at https://github.com/mysociety/alaveteli/issues/7438 ), and in one case to make a proposal to rescue, and republish, a FOI response. 

Ideally we'd be tracking this activity publicly, and automatically via https://github.com/mysociety/alaveteli/issues/1203
